### DISCUSSION TITLE
[Feature] Apple TV app?

### DISCUSSION THREAD

**Discussion by peros550** [immich-app/immich#14029] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

One of Apple TV's best features is the support for Photos app. 
Since Immich is one best candidates for Apple's Photos app replacement, it would make perfect sense to askwhether an Apple TV app would be somewhere down the roadmap. 

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by CiNcH83** [comment]
Is the Photos app on Apple TV any good? Will need to try that...

I am still of the opinion that controlling any TV is a pain. Mobile devices on the other hand have become brilliant navigation devices even for large collections of items which is why I would prefer casting.

**Comment by bo0tzz** [comment]
#3892
